projects/packages/my-jetpack/_inc/components/plans-section/style.module.scss

```diff
@@ -1,4 +1,7 @@
 .container {
+	--jp-black: #000;
```

Comment:
You don't need to introduce this one, it comes from `base-styles`.

Reply:
nice. let me update it

Reply:
addressed here https://github.com/Automattic/jetpack/pull/23007/commits/09e58819feb7c2e3397c04a25f69d6c49b0cc13e